benchmark/Manifest.toml

```diff
@@ -1,25 +1,35 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.8.5"
+julia_version = "1.9.0"
```

Comment:
You also want to change https://github.com/Team-RADDISH/ParticleDA.jl/blob/a0e7ea754bd8a9628df2dfaa6b0ce8b4b2a671dd/.github/workflows/benchmark.yml#L14 to `1.9.0` for consistency (side note, I'm not 100% happy with the benchmark setup in this package, results are often _weird_)

Reply:
Ah thanks for the pointer!